wgs/wg-automl/charter.md

```diff
@@ -1,39 +0,0 @@
-# WG AutoML Charter
```

Comment:
Why `charter.md` is deleted?

Reply:
The charter is now auto-generated from the `wgs.yaml` and placed into `wg-*/README.md`

Reply:
I mis-understood the charter part. I'm adding them back under `wg-*/charter.md`